Darians mupotsal

mupotsal opened the pull request:
Have we done the right thing for you to accept our code.

mupotsal committed:
WE changed the sleep delays into a loop

mupotsal committed:
WE changed the sleep delays into better loops. We also added and commented on functions.

mupotsal committed:
WE changed the sleep delays into better loops. We also added and commented on functions.

mupotsal committed:
WE changed the sleep delays into better loops. We also added and commented on functions.
We have added the google document link.

emmet0r reviewed (changes requested):
You're on the right track! A few small things:
- Can you add back in the comment line with the original author (Sandesh)?
- You don't need to define or call main, since those things should already exist at the bottom of the file.
- At first glance, I'm wondering if your `kill_if_dead()` function is in the right place... 🤔Have you tested your code with all the possible user inputs/options?

mupotsal commented:
Let me do it right now,Thank you

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________
From: Emily Lovell <notifications@github.com>
Sent: Sunday, February 10, 2019 6:11 PM
To: spring-2019-csc-226/t04-master
Cc: Liberty Mupotsa; Author
Subject: Re: [spring-2019-csc-226/t04-master] Darians mupotsal (#13)

[EXTERNAL SENDER]

@emmet0r requested changes on this pull request.

You're on the right track! A few small things:

  *   Can you add back in the comment line with the original author (Sandesh)?
  *   You don't need to define or call main, since those things should already exist at the bottom of the file.
  *   At first glance, I'm wondering if your kill_if_dead() function is in the right place... 🤔Have you tested your code with all the possible user inputs/options?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/spring-2019-csc-226/t04-master/pull/13#pullrequestreview-201927999>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AsY-O4qTpJPVMlEN2urB72mSUMuT3TCmks5vMKcDgaJpZM4axdGn>.

mupotsal committed:
WE changed the sleep delays into better loops. We also added and commented on functions.
We have added the google document link.
Put the if_dead at the correct place and put back Sanders.

mupotsal commented:
We have made the adjustments and we think we have fixed the bug.

Thank you!

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________
From: Liberty Mupotsa <mupotsal@berea.edu>
Sent: Sunday, February 10, 2019 6:12 PM
To: spring-2019-csc-226/t04-master; spring-2019-csc-226/t04-master
Cc: Author
Subject: Re: [spring-2019-csc-226/t04-master] Darians mupotsal (#13)

Let me do it right now,Thank you

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________
From: Emily Lovell <notifications@github.com>
Sent: Sunday, February 10, 2019 6:11 PM
To: spring-2019-csc-226/t04-master
Cc: Liberty Mupotsa; Author
Subject: Re: [spring-2019-csc-226/t04-master] Darians mupotsal (#13)

[EXTERNAL SENDER]

@emmet0r requested changes on this pull request.

You're on the right track! A few small things:

  *   Can you add back in the comment line with the original author (Sandesh)?
  *   You don't need to define or call main, since those things should already exist at the bottom of the file.
  *   At first glance, I'm wondering if your kill_if_dead() function is in the right place... 🤔Have you tested your code with all the possible user inputs/options?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/spring-2019-csc-226/t04-master/pull/13#pullrequestreview-201927999>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AsY-O4qTpJPVMlEN2urB72mSUMuT3TCmks5vMKcDgaJpZM4axdGn>.
